assets/javascripts/discourse/components/query-result.js.es6

```diff
@@ -139,6 +140,61 @@ const QueryResultComponent = Ember.Component.extend({
     return transformedRelTable(groups);
   },
 
+  @computed(
+    "rows.[]",
+    "content.colrender.[]",
+    "content.result_count",
+    "colCount"
+  )
+  canShowGraph(rows, colrender, result_count, colCount) {
+    const hasTwoColumns = colCount === 2;
+    const secondColumnContainsNumber =
+      result_count > 0 && typeof rows[0][1] === "number";
+    const secondColumnContainsId = colrender[1] !== undefined;
```

Comment:
de we explicitly need to check for undefined here ?

Reply:
It would work without an explicit check.

But I feel that with an explicit check it's easier to understand what's going on here. But I can change this if it is more idiomatic for the Discourse codebase:
```
const secondColumnContainsId = colrender[1];

return (
  hasTwoColumns && secondColumnContainsNumber && !secondColumnContainsId
);
```